content/enterprise/operational-modes/index.mdx

```diff
@@ -0,0 +1,162 @@
+| [Mounted Disk](#mounted-disk)           | Stored in a directory on the instance backed by user-managed persistent storage. | Stored in a directory on the instance backed by user-managed persisent storage. | Stored in the PostgreSQL Database unless using [External Vault](#external-vault). | Stored in a Docker volume on the instance. |
+| [External Services](#external-services) | Stored in an external user-managed PostgreSQL database.                          | Stored in an external user-managed object storage location.                     | Stored in the PostgreSQL Database unless using [External Vault](#external-vault). | Stored in a Docker volume on the instance. |
+
+## Mounted Disk
+
+The Mounted Disk operational mode stores the PostgreSQL Database and Object Storage data in a directory on the instance. It is expected that this directory is backed by persistent storage that can be mounted to the instance. This allows you to backup and restore the PostgreSQL Database or Object Storage data independently of the Terraform Enterprise instance.
+
+You are responsible for providing and maintaining the persistent storage.
+
+Configuration data is stored on a Docker volume on the instance. You can retrieve this information using `replicatedctl app-config export`.
+
+### When to Use Mounted Disk
+
+Mounted Disk is recommended when:
+
+- Running Terraform Enterprise in an environment that does not have an external PostgreSQL server or S3-compatible object storage.
+- You have experience managing persistent storage such as an AWS EBS volume, an Azure Managed Disk, a Google Cloud Persistent Disk, an iSCSI location, etc.
+- Are familiar with using tools like `cp`, `scp`, and `rsync` to backup and restore data.
+
+## External Services
+
+The External Services operational mode stores the PostgreSQL Database and Object Storage data on an external PostgreSQL server and S3-compatible object storage. This allows you to backup, restore, and scale the PostgreSQL Database or Object Storage independently of the Terraform Enterprise instance.
+
+You are responsible for providing and maintaining the external PostgreSQL server and S3-compatible object storage.
+
+Configuration data is stored on a Docker volume on the instance. You can retrieve this information using `replicatedctl app-config export`.
+
+### When to Use External Services
+
+External Services is recommended when:
+
+- Running Terraform Enterprise in environment that has an external PostgreSQL server or S3-compatible object storage.
+- You have experience managing a PostgreSQL server such as Amazon RDS for PostgreSQL, Google Cloud SQL for PostgreSQL, Azure Database for PostgreSQL, etc.
+- You have experience managing an S3-compatible object storage location such as AWS S3, Azure Blob Storage, Google Cloud Storage, MinIO, etc.
+- Want to use native backup and restore features provided by the external PostgreSQL server and S3-compatible object storage.
+
+## External Vault
```

Comment:
This is not currently on the pre-install checklist (there we have active/active, mounted disk, and external services). Should it be? I'm confused about the actual complete list of all the modes (is active/active one? Is External vault one?). Can we make sure everything is consistent across these pages?  Please forgive me if this is my fault when I did the rework of the sidebar!

rAnd then, I'm wondering if we can cut some of this because we have this nice new page now :) : https://www.terraform.io/enterprise/requirements/data-storage/vault

Reply:
External Vault is not an operational mode in itself. It can be used across the different operational modes. I removed the information and updated the new page content.